allennlp/data/dataset_readers/semantic_role_labeling.py

```diff
@@ -14,6 +15,54 @@
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
+START_TOKEN = "[CLS]"
+SEP_TOKEN = "[SEP]"
+
+def _convert_tags_to_wordpiece_tags(tags: List[str], offsets: List[int]):
```

Comment:
I would add a docstring explaining why this is here and mentioning that it will only get used if you're using BERT